plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/conf/hydra/launcher/ray_aws.yaml

```diff
@@ -4,6 +4,9 @@ hydra:
     cls: hydra_plugins.hydra_ray_launcher.ray_aws_launcher.RayAWSLauncher
     params:
       stop_cluster: False
+
+      # assuming you are running from plugin root root.
+      code_dir: example
```

Comment:
This is the general plugin config, what is example and why should it be copied by default?
(Should this be in the example app config)?